integrations/slack/src/router.ts

```diff
@@ -45,6 +56,67 @@ export const handleFetchEvent: FetchEventCallback = async (request, context) =>
         ].join(' '),
     );
 
+    async function verifyIntegrationSignature(
```

Comment:
Not for this PR but I think this is dupped across multiple integrations. It would be nice to have something factored out (along with some tests) into a shared package (maybe the runtime).